flow/Profiler.actor.cpp

```diff
@@ -33,7 +33,7 @@
 #include "flow/Platform.h"
 #include "flow/actorcompiler.h" // This must be the last include.
 
-extern volatile thread_local int profilingEnabled;
+extern volatile int profilingEnabled;
 
```

Comment:
I'm not sure if this change really works. We don't use this profiler, but probably to be safest we could have two enabled variables, one thread local for here and another that isn't thread local for the slow task profiler.